### PR TITLE
[Magic WAN] Adding On-ramp tutorial

### PR DESCRIPTION
**Pull request by dcpena**
- Added tutorial on how to set up a GRE on-ramp
- Removed references to LOA and IRR

Addresses [PCX-2741](https://jira.cfops.it/browse/PCX-2741).

### CODE REVIEW

File: products/magic-wan/src/content/get-started/index.md [hunk 1]
```diff
@@ -24,6 +24,6 @@ After Cloudflare stages the tunnels, Cloudflare validates tunnel connectivity, t
 
 ## 4. Go live
 
-During this step, provide Cloudflare with the LOA for your prefixes and the exchange GRE tunnel information. During this call, you should also [review your Firewall configuration](https://developers.cloudflare.com/magic-firewall/).
+During this step, provide Cloudflare with the exchange GRE tunnel information. During this call, you should also [review your Firewall configuration](https://developers.cloudflare.com/magic-firewall/).
```
Comment (marciocloudflare) [inline products/magic-wan/src/content/get-started/index.md]: Is there a way we can replace one of the `During`? Both sentences start with it, but without context I don't know if we can :)

File: products/magic-wan/src/content/on-ramp/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Contact your account manager to enable Teams Gateway on your account.
+
+### 2. Set up your client
+
+First, reserve an IP for the GRE tunnel and then create a VM instance. The example below uses the Google Cloud Platform, but you can choose what to connect.
+
+**Reserve an IP for the GRE tunnel**
+
+`gcloud compute addresses create $ADDRESS_NAME   --region=$REGION`
```
Comment (marciocloudflare) [inline products/magic-wan/src/content/on-ramp/index.md]: This seems to have an extra space or 2 before `--region`. Does it?

**Review by marciocloudflare — APPROVED** [review]

File: products/magic-wan/src/content/get-started/index.md (hunk 1 above)
Comment (dcpena) [reply products/magic-wan/src/content/get-started/index.md]: Good catch, thanks! Fixed

File: products/magic-wan/src/content/on-ramp/index.md (hunk 2 above)
Comment (dcpena) [reply products/magic-wan/src/content/on-ramp/index.md]: Fixed

**Pull request merged**